task.py

```diff
@@ -0,0 +1,35 @@
+'''
+Task 1. Who are you?

```

Comment:
You've arrived to the new planet of **Ultra-Super-Coders**. 

Your first task is a proper greeting.

In order to be accepted in society, you need to properly introduce yourself. You need to declare yourself.


To do this, write your first official **introductory program**, which will display your ```first name``` and ```last name```, ```nickname```,
```email```, as well as ```date of birth``` and ```native language```.

Of course, the **input** data:
* ```first name```
* ```last name```
* ```nickname```
* ```email```
* ```date of birth```
* ```native language```

The output should be made according to the **template**:
```
Hi coders! I want to join your community!
My name is [first name] [last name]!
My native language is [native language]! And I'm proud of it!
I was born [date of birth]. Not bad, right?
You can contact me like [nickname] or [email]. 
I think we will be friends :) 
And I wish you a great time on this planet!
```

**Example**:
**Input data**:
```
Rick
Sanchez
RickyRick
whereIsMyMorty@rick.com
01.01.1949 
English    
```
**Output data**:
```
Hi coders! I want to join your community!
My name is Rick Sanchez!
My native language is English  ! And I'm proud of it!
I was born 01.01.1949 . Not bad, right?
You can contact me like RickyRick or whereIsMyMorty@rick.com.
I think we will be friends :) 
And I wish you a great time on this planet!
```